src/main/java/com/github/valfirst/jbehave/junit/monitoring/JUnitReportingRunner.java

```diff
@@ -32,6 +32,7 @@
 	private int numberOfTestCases;
 	private Description rootDescription;
 	private ConfigurableEmbedder configurableEmbedder;
+	public String unusedVariable;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make unusedVariable a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.oss.wandera.net/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)